src/fastga/models/aerodynamics/components/wing/compute_cl_wing.py

```diff
@@ -0,0 +1,83 @@
+"""Estimate wing lift coefficient."""
```

Comment:
Here it should be clarified that this computation of the wing lift coefficient is only used to estimate the derivative with respect to the roll rate. For the estimation during the mission it is very different. Maybe even in the docstring directly